AutomationStudio4.gitignore

```diff
@@ -0,0 +1,36 @@
+# gitignore template for B&R Automation Studio (AS) 4
+# website: https://www.br-automation.com/en-us/products/software/automation-software/automation-studio/
+
+# AS temporary directories
+Binaries/
+Diagnosis/
+Temp/
+TempObjects/
+
+# AS transfer files
+*artransfer.br
+*arTrsfmode.nv
+
+# 'ignored' directory
+ignored/
+
+# ARNC0ext
+*arnc0ext.br
+
+# AS File types
+*.bak
+*.isopen
+*.DS_Store
```

Comment:
We don't accept OS specific rules in template files unless they're added to the OS specific template. See [macOS.gitignore](https://github.com/github/gitignore/blob/7b22f8ab6c85b4ef1469d72a8ba96462e2a44853/Global/macOS.gitignore) for an example. Could you remove the OS specific rules?

Reply:
Yes! Removed.